app/client/keybase/keyvault.go

```diff
@@ -0,0 +1,296 @@
+package keybase
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/pokt-network/pocket/shared/crypto"
+)
+
+// VaultKeybase implements the Keybase interface using HashiCorp Vault
+type VaultKeybase struct {
+	client *api.Client
+	mount  string
+}
+
+// NewVaultKeybase returns a new instance of VaultKeybase
+func NewVaultKeybase(client *api.Client, mount string) *VaultKeybase {
+	return &VaultKeybase{
+		client: client,
+		mount:  mount,
+	}
+}
+
+// Create new keypair entry in Vault
+func (vk *VaultKeybase) Create(passphrase, hint string) error {
+	keyPair, err := crypto.CreateNewKey(passphrase, hint)
+	if err != nil {
+		return err
+	}
+
+	keyPairBytes, err := json.Marshal(keyPair)
+	if err != nil {
+		return err
+	}
+
+	data := map[string]interface{}{
```

Comment:
SInce you're using this in a few places in the same file, let's create a small local struct near the top.